src/projectscene/qml/Audacity/ProjectScene/clipsview/ClipItem.qml

```diff
@@ -5,6 +5,7 @@ import Muse.Ui
 import Muse.UiComponents
 import Muse.GraphicalEffects
 
+import Audacity.Preferences
```

Comment:
There is a dependency on the appshell module in projectscene, this is not allowed. appshell knows about all modules, but modules should not know about appshell.

If you need some setting from the preferences model, put it in the projectscene(?) configurator and reuse it in some clip model.